### ISSUE TITLE
Monochrome icon for macOS

### ISSUE THREAD

**Issue by hovancik** [Tribler/tribler#2899]
Hi guys, 

I would like to make a small contribution: create monochrome icon for Tribler for macOS: 

![tribler-macos](https://cloud.githubusercontent.com/assets/4745859/24906661/f3584e1e-1eb8-11e7-9729-de9b3f3f8da4.png)

Why? 

![screen shot 2017-04-11 at 13 29 03](https://cloud.githubusercontent.com/assets/4745859/24907075/e3c42296-1eba-11e7-8dcb-3e78d59436f2.png)

Would a PR be accepted? I guess `devel` branch?

**Comment by devos50** [comment]
Hi,

Thank you for your contribution! I agree a black/white logo is better in some occurrences (my task bar is also mostly gray).

It would be nice if we have a way to toggle between the colored and black/white logo. Could you create a PR where you make this change? The logo itself is loaded in code here: https://github.com/Tribler/tribler/blob/devel/TriblerGUI/tribler_window.py#L141

Thanks!

**Comment by hovancik** [comment]
Thanks! I'll try and make a PR.

**Comment by hovancik** [comment]
@devos50 can you please have a look? I have added the option to menu, but I can't figure out how to access the settings at point of creating the tray.

**Comment by KeithWM** [comment]
Has this fix somehow been lost in subsequent changes? I was looking for this feature and can see the PR has been merged, but the code at the time of the PR seems to bare little resemblence to the current state. Has there been a major overhaul or rewriting that has resulted in this feature being lost?

**Comment by qstokkink** [comment]
@KeithWM correct. This should be reasonably easy and quick to add back, though. I'll give this a go for inclusion in 8.3.0.

**Comment by KeithWM** [comment]
Thanks! Let me know if I can help with a review or anything.

**Comment by qstokkink** [comment]
I tinkered a bit and I made it so that you can adapt the tray icon color to whatever you like:

<img width="182" height="288" alt="Image" src="https://github.com/user-attachments/assets/f4f4e9c9-637c-4b15-a9e2-db6157b57ec7" />
